src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.html

```diff
@@ -1,5 +1,5 @@
 <exui-task-list
   [taskServiceConfig]="taskServiceConfig" [tasks]="tasks" [fields]="fields"
-  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage"
+  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage" [sourceColumn]="sourceColumn" [matchValue]="matchValue"
```

Comment:
I don't see why the `sourceColumn` and `matchValue` properties are part of the `TaskListComponent`. That doesn't seem right to me.

Reply:
Changed as per Ronald's new info